eclass/linux-mod-r1.eclass

```diff
@@ -131,6 +131,15 @@ IDEPEND="
 	sys-apps/kmod[tools]
 "
 
+if [[ ${MODULES_IUSE_INITRAMFS} ]]; then
+	if [[ ${MODULES_IUSE_INITRAMFS} == default_enabled ]]; then
```

Comment:
Wonder if should just do this like `MODULES_OPTIONAL_IUSE` so it'd be more uniform rather than have two ways to handle these. Would also allow to rename the IUSE if it was ever needed (doubtful, but well).

aka rename the variable to match and do `MODULES_INITRAMFS_IUSE=+initramfs` if want it default on, can be added as-is to IUSE, albeit would need to repeat `${MODULES_INITRAMFS_IUSE#+}' in a few spots for use/depend.

Reply:
Fixed!